apps/web/src/components/Settings/BasicInfo.tsx

```diff
@@ -278,7 +278,7 @@ const BasicInfo = ({ channel }: Props) => {
           }
           className="object-cover object-center w-full h-48 bg-white rounded-xl md:h-56 dark:bg-gray-900"
           draggable={false}
-          alt="Cover Image"
+          alt={channel.handle + ' Cover'}
```

Comment:
can we use the template literal for this one?

Reply:
@sasicodes  done